ui/src/pages/ResultsPage.tsx

```diff
@@ -4916,19 +4937,45 @@ const InventoryTabContent = (props: {
 				</Toolbar>
 				{scan.results?.inventory?.base_images ? (
 					<EnhancedTable
-						columns={columns}
-						rows={rows}
+						columns={baseImageColumns}
+						rows={baseImageRows}
 						defaultOrderBy="image"
 						menuOptions={{
 							exportFile: "scan_images",
 							exportFormats: ["csv", "json"],
-							exportData: exportData,
+							exportData: baseImageExportData,
 						}}
 					/>
 				) : (
 					<NoResults title={i18n._(t`No base images found`)} />
 				)}
 			</Paper>
+			<Paper square className={classes.paper}>
+				<Toolbar>
+					<Typography variant="h6" id="cicd-tools-title" component="div">
+						<Trans>Potential CI/CD Tools</Trans>
+						{baseImageRows && (
+							<CustomCopyToClipboard
+								copyTarget={cicdToolsRows.map((data) => data.tool).join(", ")}
+							/>
+						)}
+					</Typography>
+				</Toolbar>
+				{(scan.results_summary?.inventory?.cicd_tools || 0) > 0 ? (
+					<EnhancedTable
+						columns={cicdToolsColumns}
+						rows={cicdToolsRows}
+						defaultOrderBy="tool"
+						menuOptions={{
+							exportFile: "cicd_tools",
+							exportFormats: ["csv", "json"],
+							exportData: getCICDRows,
```

Comment:
I noticed the exported JSON has the filenames separated by commas instead of being an array -- anything we can do about that?

```json
{
  "tool": "Jenkins",
  "files": "Jenkinsfile,build/Jenkinsfile"
}
```

Reply:
This was intended to clean output for CSVs, but it turns out we convert arrays to escaped comma-separated strings automatically. Good catch